README.md

```diff
@@ -24,17 +24,35 @@ To use the latest release for local testing run the following steps. dotnet does
 2. run `docker-compose up`
 3. Access the API from your browser [http://localhost:5023/playground](http://localhost:5023/playground)
 
-## using the dotnet command-line tools
+## Work on the code
+
+If you work on the code you probably want to run the Api and check if your changes work as expected.
 
 **Prerequisites**
 - dotnet core 2.1 SDK installed
   
-**Setup**
+**Build and Test the Api**
+
+1. Open command-line and navigate to the root of the repository
+2. run `build .ps1` on a windows system or `build.sh` on linux
```

Comment:
Powershell exists on every system now <3 
But good that you don't have to have it installed on Unix.

Reply:
Nevermind. The ps1 build relies on an exe.